### ISSUE TITLE
Add aarch64 binaries to releases?

### ISSUE THREAD

**Issue by adamrankin** [commontk/AppLauncher#120]
Hello,

Is it possible to include a binary build for aarch64 into the releases?

**Comment by jcfr** [comment]
That should indeed be possible. We would be happy to review contributions.

Here are the steps:

* Create a static build of Qt and document how to create and update such a build. For each platform we currently support, we download a static build of Qt. See details at https://github.com/jcfr/qt-static-build
* Update continuous integration to build for aarch64. May be we can re-use some of the work done to support building cmake python wheel for aarch64 (see https://github.com/scikit-build/cmake-python-distributions/blob/master/.travis.yml)

**Comment by adamrankin** [comment]
The static build of Qt is 4.8.6. Did you want to continue using that version?

**Comment by jcfr** [comment]
> The static build of Qt is 4.8.6. Did you want to continue using that version?

If you already have a Qt build for aarch64 and it 4.8.6, that is a good starting point.

What do you think of creating a release called `applauncher-4.8.6-aarch64` in https://github.com/jcfr/qt-static-build/releases and update the [README](https://github.com/jcfr/qt-static-build/blob/master/README.md) ? 

If that sounds good, I will create the release and grant access so that we you can update the release description, upload file and describe how it was generated.

**Comment by adamrankin** [comment]
I'm still trying to wrap my head around all the automated build system stuff. Would this not be the same as the linux docker image, just with a different base image?

**Comment by adamrankin** [comment]
Ok, @jcfr I've made a docker image with Qt 5.12.10 built. Should I zip up the Qt build folder inside and upload it to a release?

**Comment by jcfr** [comment]
> made a docker image with Qt 5.12.10 built

This is great.

I suggest the following next steps:
* check that you can run the test from the build tree of the AppLauncher
* check that the binary created can be run on the target platform (you can do `make package` to first create an archive)

**Comment by adamrankin** [comment]
Good call. libpng linking issues! Will update when I have a working package.

**Comment by adamrankin** [comment]
Should I redo my docker image to be based on a dockcross image? With my current docker image it would require an arm64 host to build the image

**Comment by adamrankin** [comment]
@jcfr At what point is it reasonable to assume that some libraries will be on most target machines? My static build is looking for pthreads to link to. How far down the chain of libraries do I build staticly?
